composer.json

```diff
@@ -1,42 +1,60 @@
 {
-    "name": "strictphp/http-clients",
-    "type": "project",
-    "description": "arious http client implementations for better developer and devops experience.",
-    "keywords": ["laravel", "php", "strictphp", "http-client"],
-    "license": "MIT",
-    "homepage": "https://github.com/strictphp/http-clients",
-    "authors": [
-        {
-            "name": "Martin Kluska",
-            "email": "martin@kluska.cz"
-        },
-        {
-          "name": "Milan Matějček",
-          "email": "milan.matejcek@gmail.com",
-          "role": "Developer"
-        }
-    ],
-    "require": {
-        "php": ">=8.1"
+  "name": "strictphp/http-clients",
+  "type": "project",
+  "description": "arious http client implementations for better developer and devops experience.",
+  "keywords": [
+    "laravel",
+    "php",
+    "strictphp",
+    "http-client"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/strictphp/http-clients",
+  "authors": [
+    {
+      "name": "Martin Kluska",
+      "email": "martin@kluska.cz"
     },
-    "require-dev": {
-       
-    },
-    "scripts": {
-       
-    },
-    "autoload": {},
-    "autoload-dev": {},
-    "extra": {
-        "laravel": {
-            "providers": []
-        }
-    },
-    "config": {
-        "optimize-autoloader": true,
-        "preferred-install": "dist",
-        "sort-packages": true
-    },
-    "minimum-stability": "dev",
-    "prefer-stable": true
+    {
+      "name": "Milan Matějček",
+      "email": "milan.matejcek@gmail.com",
+      "role": "Developer"
+    }
+  ],
+  "require": {
+    "php": ">=8.1",
+    "guzzlehttp/psr7": "^2.5",
+    "illuminate/filesystem": "^9.0 || ^10.0",
```

Comment:
chceme toto pak nahradit nebo necháme laravel implementaci?

Reply:
Používám jen interface, jsem pro to nechat.